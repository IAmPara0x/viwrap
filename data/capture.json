{"_captureTermSize":[64,238],"_startTime":1.68284715280700777e9,"_stdinContents":[{"_inputAt":2.898251438,"_inputContents":"1"},{"_inputAt":0.118754977,"_inputContents":"0"},{"_inputAt":0.614604962,"_inputContents":"\""},{"_inputAt":0.248989854,"_inputContents":"\u001b"},{"_inputAt":0.477050719,"_inputContents":"I"},{"_inputAt":0.344239845,"_inputContents":"\""},{"_inputAt":0.327745353,"_inputContents":"\n"},{"_inputAt":2.953899384,"_inputContents":"\u001b"},{"_inputAt":0.200121568,"_inputContents":"k"},{"_inputAt":0.773877265,"_inputContents":"I"},{"_inputAt":1.176093658,"_inputContents":"a"},{"_inputAt":0.185923696,"_inputContents":" "},{"_inputAt":0.246057098,"_inputContents":"="},{"_inputAt":0.375958695,"_inputContents":" "},{"_inputAt":0.327898719,"_inputContents":"\n"},{"_inputAt":1.440281422,"_inputContents":"e"},{"_inputAt":0.183815174,"_inputContents":"x"},{"_inputAt":4.0061436e-2,"_inputContents":"i"},{"_inputAt":0.184858837,"_inputContents":"t"},{"_inputAt":0.127029227,"_inputContents":"9"},{"_inputAt":8.8169673e-2,"_inputContents":")"},{"_inputAt":0.263948771,"_inputContents":"\n"},{"_inputAt":0.624118461,"_inputContents":"e"},{"_inputAt":0.183754783,"_inputContents":"x"},{"_inputAt":3.9045814e-2,"_inputContents":"i"},{"_inputAt":0.129092526,"_inputContents":"t"},{"_inputAt":0.144007085,"_inputContents":"("},{"_inputAt":6.4004461e-2,"_inputContents":")"},{"_inputAt":0.272018731,"_inputContents":"\n"}],"_stdoutContents":["Python 3.10.9 (main, Dec 19 2022, 17:35:49) [GCC 12.2.0] on linux\r\nType \"help\", \"copyright\", \"credits\" or \"license\" for more information.\r\n",">>> ","1","0","\"","\u0008\u001b[K\u0008\u001b[K\u0008\u001b[K\"10\"","\r\n","'10'\r\n>>> ","\"10\"","\u0008\u001b[K\u0008\u001b[K\u0008\u001b[K\u0008\u001b[Ka\"10\"","\u0008\u001b[K\u0008\u001b[K\u0008\u001b[K\u0008\u001b[K \"10\"","\u0008\u001b[K\u0008\u001b[K\u0008\u001b[K\u0008\u001b[K=\"10\"","\u0008\u001b[K\u0008\u001b[K\u0008\u001b[K\u0008\u001b[K \"10\"","\r\n",">>> ","e","x","i","t","9",")","\r\n  File \"<stdin>\", line 1\r\n    exit9)\r\n         ^\r\nSyntaxError: unmatched ')'\r\n",">>> ","e","x","i","t","(",")","\r\n"]}