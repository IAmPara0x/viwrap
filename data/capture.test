Python 3.10.9 (main, Dec 19 2022, 17:35:49) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> [?25ll[?25h[?25la[?25h[?25ll[?25h[?25la[?25h
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'lala' is not defined
>>> [?25h[?25ll[?25h[?25lw[?25h[?25ll[?25h[?25lw[?25h
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'lwlw' is not defined
>>> [?25h[?25ll[?25h[?25la[?25h[?25ll[?25h[?25la[?25h
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'lala' is not defined
>>> [?25h[?25le[?25h[?25lx[?25h[?25li[?25h[?25lt[?25h[?25l([?25h[?25l)[?25h
