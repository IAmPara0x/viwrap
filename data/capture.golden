Python 3.10.9 (main, Dec 19 2022, 17:35:49) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> 10"[K[K[K"10"
'10'
>>> "10"[K[K[K[Ka"10"[K[K[K[K "10"[K[K[K[K="10"[K[K[K[K "10"
>>> exit9)
  File "<stdin>", line 1
    exit9)
         ^
SyntaxError: unmatched ')'
>>> exit()
